aten/src/ATen/FunctionalizeFallbackKernel.cpp

```diff
@@ -34,6 +34,15 @@ namespace {
           auto t_new = c10::IValue(at::functionalization::impl::from_functional_tensor(tensors));
           (*stack)[arguments_begin + idx] = t_new;
         }
+      } else if (ivalue.isOptionalTensorList()) {
+        any_tensor_inputs = true;
+        auto opt_tensors = ivalue.toOptionalTensorList();
+        if (at::functionalization::impl::isFunctionalTensor(opt_tensors)) {
```

Comment:
Is there an invariant that either all or none of these must be functional tensors, or can there be some hybrid mix?